sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -971,6 +974,44 @@ def test_groupby_sum_min_count(self):
 
     self._run_test(lambda df: df.groupby('group').sum(min_count=2), df)
 
+  @parameterized.expand(frames.ALL_AGGREGATIONS)
+  def test_dataframe_groupby_series(self, agg_type):
+    if agg_type == 'describe' and PD_VERSION < (1, 2):
+      self.skipTest(
+          "BEAM-12366: proxy generation of DataFrameGroupBy.describe "
+          "fails in pandas < 1.2")
+    self._run_test(
+        lambda df: df[df.foo > 40].groupby(df.group).agg(agg_type), GROUPBY_DF)
```

Comment:
Add a test where the series is an expression rather than a plain old column.

Reply:
Good point, done.